MovieApp/MovieApp/App/Features/Movies/Components/HomeComponents/SubcategoryView/SubcategoryScrollView.swift

```diff
@@ -4,9 +4,11 @@ class SubcategoryScrollView: UIScrollView {
 
     let stackViewSpacing = CGFloat(23)
 
-    var indexSelected = 0
+    var indexSelected: Int!
     var stackView: UIStackView!
 
+    var flag = 0
```

Comment:
If this flag has values 0 and 1, why can't it be a boolean? Also, this could be private and should have a more descriptive name.